### PR TITLE
Allows the HoP to open extra chaplain slots

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Allows multiple chaplains to spawn without fucking up the ticker. The later chaplains will spawn with the bible of the original one, and they spawn with their own weapon.

Maybe I'll add a new ID icon and actually change their job title if people don't hate the idea.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
an infinity of cultist-hunting assholes!

**Comment by KorPhaeron** [comment]
@ChangelingRain 

yeah but you can already do the same thing with security officers, and they even spawn with extra tasers

**Comment by ChangelingRain** [comment]
Security officers don't come with a weapon that makes them immune to a TON of cult bullshit, and while officers do have loyalty implants, you can remove implants. You can't remove inbuilt conversion immunity.

**Comment by ShadowDeath6** [comment]
this seems kinda strong but nobody wants to play a third chaplain without meta-ing so it probably(citation needed) won't be a problem???

**Comment by Durkel** [comment]
Who cares? The chaplain is the only reliable way to deconvert someone anyway. And god forbid someone be immune to the absolute powercreep cluster fuck that is blood and clock cult.

**Comment by Bawhoppen** [comment]
Will they actually be able to use their biblrs?

**Comment by PKPenguin321** [comment]
chaplain tide station wide

**Comment by JJRcop** [comment]
Maybe we should map in an additional confessional.

**Comment by Shadowlight213** [comment]
Only fine with this if they don't spawn with a nullrod.

**Comment by KorPhaeron** [comment]
Nullrods being irreplaceable is silly anyway given how critical they are

**Comment by Dorsisdwarf** [comment]
This is the sort of thing that needs a longer test run to see the balance implications, but I'm a sucker for chaplain gimmicks and this seems like a good way to gimmick them up

**Comment by GunHog** [comment]
Call them acolytes! The main Chaplain is the leader, and the acolytes "answer" to him!

File: code/modules/jobs/job_types/civilian_chaplain.dm [hunk 1]
```diff
@@ -34,7 +34,18 @@ Chaplain
 	if(visualsOnly)
 		return
 
+
 	var/obj/item/weapon/storage/book/bible/B = new /obj/item/weapon/storage/book/bible/booze(H)
+
+	if(ticker && ticker.Bible_deity_name)
+		B.deity_name = ticker.Bible_deity_name
+		B.name = ticker.Bible_name
+		H << "There is already an established religion onboard the station. You are an acoylte of [ticker.Bible_deity_name]. Defer to the Chaplain."
```
Comment (Tacolizard) [inline code/modules/jobs/job_types/civilian_chaplain.dm]: you spelled acolyte wrong

**Comment by Sligneris** [comment]
> If someone REALLY wants to dedicate their round to serving the chaplain, they can do so entirely within the game [going to the HoP for access] without the benefits associated with being a chaplain [particularly in games like cult].

Except that's the issue. For such a key role to be irreplaceable mechanically is kind of silly, isn't it?

**Comment by ExcessiveUseOfCobblestone** [comment]
so why not create an item similar to the lawyer badge people can use to chaplain it up instead?

**Comment by Sligneris** [comment]
Well, it takes both command opening up job slots and someone joining to fill said slots... Which you can prevent as a cultist disposing by disposing of heads of staff, HoP and Captain in particular..

It's more balanced that killing a simple greytiding memester and then having the entire rest of the round railroad in your favor because no one will ever able to replace them.

**Comment by ExcessiveUseOfCobblestone** [comment]
I would say it's in bad taste to leave it up to chance for someone to not do something that cucks antags that hard with no drawbacks or considerable amount of work involved. Cult already gets meme'd on even without a chaplain, so I also don't think simply upping the chances [so easily] that someone robust/validhunty will join as secondary chaplain positions would help.

If the meta is people opening job slots roundstart for people to join as chaplain, people will willingly wait in the lobby for a few minutes, as they already do something similar with ghost roles.

**Comment by KorPhaeron** [comment]
Again, same thing could hypothetically happen with sec spam in rev or warp ops, but you rarely see it.

Cult runs a bit slower so I guess there is more potential for it being used, but calling in a counter crusade sounds more fun than just calling the shuttle.

**Comment by Sligneris** [comment]
Yeah.

The thing with cult is that the moment chaplain goes missing, you might as well call the shuttle right away - because out of sudden, there's simply no way to stop the cult, save for slaughtering them all.

But you can't just get on with it either, because cults are typically based on stealth, raising their numbers in secret.

It would be better to just be able to get a secondary chaplain. And as you've pointed out yourself, any abuse if this feature would fade with time.

**Comment by jughu** [comment]
>chaplain is the only reliable job for deconverting
not killing cultists on spot for salt anyways

**Comment by optimumtact** [comment]
Kor pls

**Comment by KorPhaeron** [comment]
Been busy

**Comment by optimumtact** [comment]
ping for reopen when you want to continue

**Comment by KorPhaeron** [comment]
I guess there isn't any work left to be done on this but to test merge and see if anything bad/good happens

**Comment by KorPhaeron** [comment]
This is test merged but I have not actually seen it used yet

**Comment by GunHog** [comment]
It will probably be rare until powergaming Sec catches on and coaxes the Captain into opening more slots.

**Comment by optimumtact** [comment]
is it ready?

**Comment by KorPhaeron** [comment]
I still have not even seen it used once, so it's probably gonna be like GunHog said

Up to you if it's worth having on server long term to see if it's good or not

**Comment by Supermichael777** [comment]
i dont see this swinging rounds

if anything its like opening more sec slots

**Pull request merged**